src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -36,6 +37,14 @@ var _ Resource = &StatefulSetResource{}
 const (
 	redpandaContainerName		= "redpanda"
 	configuratorContainerName	= "redpanda-configurator"
+
+	userID	= 101
+	groupID	= 101
+	fsGroup	= 101
+
+	configDir	= "/etc/redpanda"
+	configuratorDir	= "/mnt/operator"
```

Comment:
It might be easier to understand if these names more closely followed the associated ENV vars, in particular, so that src, dst are obvious.

Reply:
I agree. I will align the names in the next push